docs/modules/ROOT/pages/documentation/payara-server/server-configuration/http/protocols.adoc

```diff
@@ -13,3 +13,604 @@ commands. The protocol configurations can be found in the admin console under
 
 image::http/protocol-config.png[Configuration Page Location]
 
+== Standard Options
+
+The following standard fields can be configured for protocols:
+
+
+[[configuration-security]]
+=== Security
+
+Whether security is enabled for the protocol. When disabled, all the SSL/TLS options are ignored.
+
+*Asadmin Command:*
+
+`set configs.config.server-config.network-config.protocols.protocol.${protocol-name}.security-enabled=[true/false]`
+
+== HTTP Options
+
+The following HTTP fields can be configured for protocols:
+
+
+[[configuration-http-server-name]]
+=== Server Name
+
+The name of the server. This will be sent as the `Server` header in responses.
+
+*Asadmin Command:*
+
+`set configs.config.server-config.network-config.protocols.protocol.${protocol-name}.http.server-name-enabled=value`
+
+[[configuration-http-default-virtual-server]]
+=== Default Virtual Server
+
+The default virtual server for the protocol. If multiple virtual servers are appropriate for a request,
```

Comment:
"The default virtual server used for the protocol. Establishes the request priority when multiple virtual servers are configured"